airlock/management/commands/run_file_uploader.py

```diff
@@ -151,4 +153,4 @@ def get_upload_files_and_update_request_status(release_request):
 
 
 def get_user_for_file(request_file):
-    return User(username=request_file.released_by, output_checker=True)
+    return User.objects.get(pk=request_file.released_by)
```

Comment:
~~If users are only going to be created in the db when they log in, there's a chance this could break things if an upload happens to be in progress when we switch to using the db users, and whoever released the files hasn't yet logged in. The uploader will keep trying forever, so once they do log in, it'll upload, so maybe the risk is not worth worrying about.~~
Nvm, got to the last commit now

Reply:
It's a good point though, as it could still happen before the manual back fill runs.

I may manually deploy this, just to avoid that.

Reply:
Thinking about it more, I may split this PR into two. One with the db models/migrations and the backfill, which we can deploy and run, and an another with switching to actually use it.  I think that way we can ensure nothing goes wrong?

Reply:
Ok, I have done this - this PR has been rebased and does not perform the switch, and I added a live backfill.

#837 will do the switch, once this is live and users have been populated.